### PR TITLE
image-builder: add fs type

### LINKED ISSUE #270: Error while building osbuilder on SLES
While running osbuilder on SLES 12 SP3, I am getting the following error
```
14:22:25 Successfully built 2c867e8f88aa
14:22:25 Successfully tagged image-builder-osbuilder:latest
14:22:26 [OK] init is installed
14:22:26 [OK] init is systemd
14:22:26 mount: /tmp/tmp.gdFlTY0MCs: wrong fs type, bad option, bad superblock on /dev/loop0p1, missing codepage or helper program, or other error.
14:22:26 umount: /tmp/tmp.gdFlTY0MCs: not mounted.
14:22:26 /osbuilder/image_builder.sh: line 203: [: 22G: integer expression expected
14:22:27 mount: /tmp/tmp.gdFlTY0MCs: wrong fs type, bad option, bad superblock on /dev/loop0p1, missing codepage or helper program, or other error.
14:22:27 umount: /tmp/tmp.gdFlTY0MCs: not mounted.
14:22:27 /osbuilder/image_builder.sh: line 203: [: 22G: integer expression expected
14:22:27 mount: /tmp/tmp.gdFlTY0MCs: wrong fs type, bad option, bad superblock on /dev/loop0p1, missing codepage or helper program, or other error.
14:22:27 umount: /tmp/tmp.gdFlTY0MCs: not mounted.
14:22:27 /osbuilder/image_builder.sh: line 203: [: 22G: integer expression expected
14:22:27 mount: /tmp/tmp.gdFlTY0MCs: wrong fs type, bad option, bad superblock on /dev/loop0p1, missing codepage or helper program, or other error.
14:22:27 umount: /tmp/tmp.gdFlTY0MCs: not mounted.
```
Here it is the complete debug console 
http://jenkins.katacontainers.io/job/kata-containers-tests-sles-12-master/3/

/cc @devimc and @jcvenegas

### PR DESCRIPTION
**Pull request by devimc**
Add filesystem type to format the loop device.

fixes #270

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: image-builder/image_builder.sh [hunk 1]
```diff
@@ -264,6 +264,7 @@ setup_loop_device() {
 format_loop() {
 	local device="$1"
 	local block_size="$2"
+	local fs_type="$3"
```
Comment (jodh-intel) [inline image-builder/image_builder.sh]: Ouch. How about also doing one of the following to make this more robust?:

- Checking the return value of `format_loop()`.
- Changing the default in the `case` `format_loop()` to be `die "Unsupported fs type: ${fs_type}"`

**Comment by devimc** [comment]
@jodh-intel change applied, thanks

**Pull request merged**